django/db/models/options.py

```diff
@@ -545,6 +545,8 @@ def get_field(self, field_name):
         """
         Return a field instance given the name of a forward or reverse field.
         """
+        if field_name == 'pk':
+            return self.pk
```

Comment:
I'm not sure. Thinking through how it would work before this change if `self.pk` was `None` then trying to do `_meta.pk.name` would have raised an `AttributeError`. The old code never caught anything like that.